homeassistant/components/alarm_control_panel/manual.py

```diff
@@ -186,33 +240,45 @@ def alarm_arm_custom_bypass(self, code=None):
 
     def alarm_trigger(self, code=None):
         """Send alarm trigger command. No code needed."""
-        self._pre_trigger_state = self._state
+        if not self._trigger_time_by_state[self._active_state]:
+            return
```

Comment:
Add a comment here why, ie "A trigger_time of zero disables the alarm_trigger service."